app/models/concerns/expirable.rb

```diff
@@ -4,6 +4,7 @@ module Expirable
   extend ActiveSupport::Concern
 
   def expires_at
+    return true if is_a?(Teacher)
```

Comment:
Is there a way we can return `nil` here, changing the return type from an optional date to an optional date or a boolean doesn't quite feel right.